libmscore/layout.cpp

```diff
@@ -3292,7 +3292,7 @@ System* Score::collectSystem(LayoutContext& lc)
       qreal layoutSystemMinWidth = 0;
       bool firstMeasure = true;
       bool createHeader = false;
-      qreal systemWidth = styleD(Sid::pagePrintableWidth) * DPI;
+      qreal systemWidth = style().pageOdd().paintWidth();
```

Comment:
`_pageEven` is only used for the margin values.  Both MPageLayout objects share the same QPageSize object, so the page sizes are identical in both.

Reply:
Yes, page size is the same, but system width should take margins into account, shouldn't it?

Reply:
Exactly, thus the `paintWidth()`  method which uses the `QPageLayout.paintRect()` instead of the `fullRect()`.  The paintRect excludes margins. It is the equivalent of the `Sid::pagePrintableWidth`.